UK/vATIS/UK - Scottish AC.json

```diff
@@ -2667,34 +2789,34 @@
         "transitionLevel": {
           "values": [
             {
-              "Low": 940,
-              "High": 958,
-              "Altitude": 85
+              "low": 940,
+              "high": 958,
+              "altitude": 85
             },
             {
-              "Low": 959,
-              "High": 976,
-              "Altitude": 80
+              "low": 959,
+              "high": 976,
+              "altitude": 80
             },
             {
-              "Low": 977,
-              "High": 994,
-              "Altitude": 75
+              "low": 977,
+              "high": 994,
+              "altitude": 75
             },
             {
-              "Low": 995,
-              "High": 1013,
-              "Altitude": 70
+              "low": 995,
+              "high": 1013,

```

Comment:
Seems to be an overlap here. 1013 is the high point while the next block starts at 1012